platforms/m1-macstadium/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "m1-macstadium",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "devDependencies": {
+    "prisma": "4.13.0-dev.19"
+  },
+  "dependencies": {
+    "@prisma/client": "4.13.0-dev.19"
+  }
```

Comment:
Where did dependencies come from before?
Sure this folder is/was not excluded from auto updating?

Reply:
The folders need to have a package.json for the script to find them. That said, we can remove the dependencies.

Reply:
Currently pnpm will not create a lockfile if there are no dependencies, so that is a small quirk that happens in 2-3 projects.